### ISSUE TITLE
[gexf] Add possibility to minimize attribute IDs

### ISSUE THREAD

**Issue by jacomyal** [graphology/graphology#398]
In some cases, I have very large GEXF files exported from graphology-gexf, because my long attribute labels are all duplicated for each node.

Since GEXF allows separating the ID to identify the attribute in each node from its `title`, it would be great to be able to use that, with a `titleToID` dictionary option for example.

**Comment by Yomguithereal** [comment]
Can't the `formatNode` or `formatEdge` option be used for such a purpose?

**Comment by Yomguithereal** [comment]
Ok, not entirely because you lose the label.

**Comment by Yomguithereal** [comment]
I must think harder about this because the gexf format has shortcomings related to `title`, `id` etc. and it is not always straigthforward to know which one you would want to use when parsing etc. regarding idempotency and ease of use. The parser lacks an option to tell which one to prefer btw (for now, it uses labels rather than incremental ids when found).

**Comment by jacomyal** [comment]
In my case it was such an issue that I did it with regexps. And just so you know, when I open the generated GEXF in another app that also uses `graphology-gexf`, the result is as expected, ie. the node attributes are indexed by their `title` and not their `id` in the resulting Graphology instance.

**Comment by jacomyal** [comment]
> Maybe some option that would automatically map to incremental ids for attribute names, like Gephi would be simpler than giving a map from title to ids in this case.

Agreed, what I need is just the reduced size, I don't really care of the "how".
